### ISSUE TITLE
Multiple compilation errors in backend.d.ts and validation.d.ts

### ISSUE THREAD

**Issue by mkrauss** [anttiviljami/openapi-backend#36]
There seem to be problems in backend.d.ts that cause compilation errors when using this library from Typescript.

```
/path/to/project/node_modules/openapi-backend/backend.d.ts(1,8): error TS1192: Module '"/path/to/project/node_modules/openapi-backend/node_modules/ajv/lib/ajv"' has no default export.
/path/to/project/node_modules/openapi-backend/backend.d.ts(78,13): error TS2411: Property 'notFound' of type 'Handler | undefined' is not assignable to string index type 'Handler'.
/path/to/project/node_modules/openapi-backend/backend.d.ts(79,13): error TS2411: Property 'notImplemented' of type 'Handler | undefined' is not assignable to string index type 'Handler'.
/path/to/project/node_modules/openapi-backend/backend.d.ts(80,13): error TS2411: Property 'validationFail' of type 'Handler | undefined' is not assignable to string index type 'Handler'.
/path/to/project/node_modules/openapi-backend/validation.d.ts(1,8): error TS1192: Module '"/path/to/project/node_modules/openapi-backend/node_modules/ajv/lib/ajv"' has no default export.
TypeScript: 5 semantic errors
```

**Comment by anttiviljami** [comment]
Hi @mkrauss which version of Typescript are you running?

**Comment by anttiviljami** [comment]
Could you also post your `tsconfig.json`

**Comment by mkrauss** [comment]
tsc --version
Version 3.3.3

```
{
  "compilerOptions": {
    "strict": true,
    "declaration": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "module": "CommonJS",
    "moduleResolution": "node",
    "target": "ESNext",
    "rootDir": "src",
    "outDir": "lib",
    "allowJs": true,
    "typeRoots": [
      "node_modules/@types"
    ]
  },
  "exclude": [
    "node_modules",
    "**/*.spec.ts"
  ]
}
```

**Comment by anttiviljami** [comment]
Thanks! I’ll try this out and report back.

**Comment by anttiviljami** [comment]
Hi @mkrauss! Could you upgrade to v2.1.0 and verify that your build is now working? :)

**Comment by mkrauss** [comment]
Hi! That seems to have fixed the "not assignable" error but not the "no default export" errors, so I'm getting just these now:

```
/path/to/project/node_modules/openapi-backend/backend.d.ts(1,8): error TS1192: Module '"/path/to/project/node_modules/openapi-backend/node_modules/ajv/lib/ajv"' has no default export.
/path/to/project/node_modules/openapi-backend/validation.d.ts(1,8): error TS1192: Module '"/path/to/project/node_modules/openapi-backend/node_modules/ajv/lib/ajv"' has no default export.
```

**Comment by anttiviljami** [comment]
Hi @mkrauss, released v2.1.1 now with esModuleInterop disabled. Any better?

**Comment by mkrauss** [comment]
That seems to resolve it, thank you!
